Document `axtg` command (#263)
radareorg/radare2book: src/analysis/code_analysis.md

(nofunc) 0x17838 [CODE] jae str.02x
```

> There are also some useful commands under `axt`. You can use `axtg` to generate radare2 commands which you can use to create graphs according to the XREFs.

Reviewer: ```suggestion
There are also some useful commands under `axt`. Use `axtg` to generate radare2 commands which will help you to create graphs according to the XREFs.
```

```
[0x08048320]> s main
[0x080483e0]> axtg
agn 0x8048337 "entry0 + 23"
agn 0x80483e0 "main"
age 0x8048337 0x80483e0
```

> You can also run `axt*` to split the radare2 commands and set flags on those corresponding XREFs.

Reviewer: ```suggestion
Use `axt*` to split the radare2 commands and set flags on those corresponding XREFs.
```
